WebRoot/js/zimbraMail/mail/model/ZmMailMsg.js

```diff
@@ -2153,7 +2153,8 @@ function(findHits, includeInlineImages, includeInlineAtts) {
 			if (!this.isRealAttachment(attach) ||
 					(attach.contentType.match(/^image/) && attach.contentId && attach.foundInMsgBody && !includeInlineImages) ||
 					(attach.contentDisposition == "inline" && attach.fileName && ZmMimeTable.isRenderable(attach.contentType, true) && !includeInlineAtts) ||
-					(attach.contentDisposition == "inline" && attach.contentType === "application/pdf" && attach.contentId && attach.foundInMsgBody)) {
+					(attach.contentDisposition == "inline" && attach.contentType === "application/pdf" && attach.contentId && attach.foundInMsgBody) ||
+					(attach.contentDisposition == "inline" || attach.contentDisposition == "attachment" && attach.contentType.match(/^image/) && attach.contentId && attach.foundInMsgBody)) {
```

Comment:
Just to confirm, is the two condition groups correct?
`attach.contentDisposition == "inline"`
OR
`attach.contentDisposition == "attachment" && attach.contentType.match(/^image/) && attach.contentId && attach.foundInMsgBody`

Isn't `attach.contentDisposition == "inline"` too wide? the 3rd conditions will be unnecessary...
```
3rd: (attach.contentDisposition == "inline" && attach.contentType === "application/pdf" && attach.contentId && attach.foundInMsgBody)